sdk/eventhub/azure-eventhub/azure/eventhub/aio/_producer_async.py

```diff
@@ -11,7 +11,7 @@
 from azure.core.tracing import AbstractSpan
 
 from .._common import EventData, EventDataBatch
-from .._producer import _set_partition_key, _set_trace_message
+from .._producer import _set_partition_key, _trace_batch_message
```

Comment:
is there a commit missing for `_trace_batch_message` ?

Reply:
this was adding duplicate "Azure.EventHubs.Message" spans, so needed to be deleted. just removed.